scripts/exgdas_enkf_earc.sh

```diff
@@ -103,14 +103,18 @@ if [ "${ENSGRP}" -eq 0 ]; then
 
 #--set the archiving command and create local directories, if necessary
         TARCMD="htar"
+        HSICMD="hsi"
         if [[ ${LOCALARCH} = "YES" ]]; then
             TARCMD="tar"
+            HSICMD=""
             [ ! -d "${ATARDIR}"/"${CDATE}" ] && mkdir -p "${ATARDIR}"/"${CDATE}"
         fi
 
         set +e
         ${TARCMD} -P -cvf "${ATARDIR}/${CDATE}/${RUN}.tar" $(cat "${ARCH_LIST}/${RUN}.txt")
         status=$?
+        ${HSICMD} chgrp rstprod "${ATARDIR}/${CDATE}/${RUN}.tar"
```

Comment:
This should be nested in an if block to check `CHGRP_RSTPROD` since S4 and, temporarily, Jet do not have the group.  `CHGRP_CMD` could also be used instead of `chgrp`, but that's perhaps nitpicky.